src/core/thread/dua_manager.cpp

```diff
@@ -520,6 +521,8 @@ void DuaManager::HandleDuaResponse(Coap::Message &aMessage, const Ip6::MessageIn
     OT_UNUSED_VARIABLE(aMessageInfo);
     otError error;
 
+    otLogCritDua("HandleDuaResponse: aResult=%s, code=%d", otThreadErrorToString(aResult), aMessage.GetCode());
```

Comment:
`otLogCrit` is usually used when some error happen. Use `Info` here?

Reply:
This is debug log. Will remove. Thanks.